cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -119,7 +119,7 @@ Feature: JWT Authenticator - Fetch signing key
     CONJ00011E Failed to discover Identity Provider
     """
 
-  Scenario: jwks uri configured with valid value
+  Scenario: ONYX-8703: jwks uri configured with correct value
```

Comment:
why do you change `valid` to `correct`, As far as I can see it there is not "correct" value to jwks uri, only "valid" and "invalid" values, just like the fact that there is no "wrong" value.